plugin.xml

```diff
@@ -71,7 +71,7 @@ xmlns:android="http://schemas.android.com/apk/res/android"
         </config-file>
 
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
-            <service android:name="by.chemerisuk.cordova.firebase.FirebaseMessagingPluginService">
+            <service android:name="by.chemerisuk.cordova.firebase.FirebaseMessagingPluginService" android:exported="true">
```

Comment:
```suggestion
            <service android:name="by.chemerisuk.cordova.firebase.FirebaseMessagingPluginService" android:exported="false">
```
According to code guides from google https://firebase.google.com/docs/cloud-messaging/android/client value should be `false`

Reply:
Chnaged `android:exported` to `false`